src/pretalx/schedule/models/slot.py

```diff
@@ -168,11 +193,13 @@ def id_suffix(self):
     @cached_property
     def frab_slug(self):
         title = re.sub(r"\W+", "-", self.submission.title)
+        title = title.lower()
+        for search, replace in SLUG_ALTERNATIVE_TRANSLATIONS.items():
+            title = title.replace(search, replace)
         legal_chars = string.ascii_letters + string.digits + "-"
         pattern = f"[^{legal_chars}]+"
         title = re.sub(pattern, "", title)
-        title = title.lower()
-        title = title.strip("_")
+        title = title.strip("-")
         return f"{self.event.slug}-{self.submission.pk}{self.id_suffix}-{title}"
 
```

Comment:
We should strip this string of `-` too, or just return the plain string in case the title is empty.